src/SonarScanner.MSBuild.PreProcessor/Resources.resx

```diff
@@ -351,4 +351,7 @@ Use '/?' or '/h' to see the help message.</value>
   <data name="MSG_Processing_PullRequest_RequestPrepareRead" xml:space="preserve">
     <value>Incremental PR Analysis: Requesting 'prepare_read' from {0}</value>
   </data>
+  <data name="MSG_DotNetAnalyzersNotFound" xml:space="preserve">
```

Comment:
I think the convention is to use `ERR_` as prefix, when the message is of a blocking error.